ftw/file/__init__.py

```diff
@@ -1,9 +1,12 @@
 """Main product initializer
 """
 from zope.i18nmessageid import MessageFactory
+from ZPublisher.BaseRequest import BaseRequest
+
 
 _ = fileMessageFactory = MessageFactory('ftw.file')
+BaseRequest.maybe_webdav_client = False
```

Comment:
1) This will be applied to production too, probably not what you want.
2) I don't understand why you have to do that, but I'd awoid changing that; I have a bad gut feeling here

Reply:
Yeah, I'll remove that - I wasn't thinking it was a good idea, but wanted to see if it fixed the tests.

As Maethu said in the daily - he has 380k of sentry errors because Zope doesn't handle HEAD requests [before version 4 at least]

Perhaps I should just remove the line `browser.default_driver = LIB_REQUESTS` and let it run as the default driver.  But arguably I should remove the test as it doesn't simulate end to end requests - which fail?

Reply:
But I think ftw.file fixed this for requests on files?? at least the test name suggests that.

Reply:
Like I said in the meeting. If the change does not have a effect on the test, you should be able to remove those lines, instead of changing them. If the test fails afterwards then either there was a side effect, or it's actually needed.

Reply:
Like I said in the meeting. If the change does not have a effect on the test, you should be able to remove those lines, instead of changing them. If the test fails afterwards then either there was a side effect, or it's actually needed.

Reply:
Ok, I've trimmed this down to just removing the superfluous code.  The tests pass on Plone 4.3 and I don't see that they will have problems on 5.1.

However, I think test_head_request is misleading as HEAD requests don't work on ftw.file (due to this Zope issue). Using LIB_REQUESTS would show this up.

For example:

```
$ curl --head https://www.zg.ch/behoerden/regierungsrat/strategie/downloads-strategie-2019-2026/legislaturziele-2019-2022.pdf/@@download/file/Legislaturziele%202019_2022.pdf
HTTP/1.1 500 Internal Server Error
```

But I think this is one for another day